docs/MatrixOne/Overview/matrixOne-architecture-design.md

```diff
@@ -82,7 +82,7 @@ The computing node (CN) is a component of Matrixone that accesses user requests
 - Pipeline, which parses the logical plan, converts the logical plan into an actual execution plan and then runs the execution plan through Pipeline.
 - Disttae, responsible for specific read and write tasks, including synchronizing Logtail from DN and reading data from S3, and sending the written data to DN.
 
-### **Streeam Engine
+### **Stream Engine**
 
 Coming soon...
```

Comment:
"Coming soon" is not a friendly expression. Please look at the suggestion of #146